bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
+        time_t const tm = system_clock::to_time_t(system_clock::now()
+            + duration_cast<system_clock::duration>(pt - lt::clock_type::now()));
```

Comment:
> do you understand why this isn't causing test failures on travis?

Afaik the travis builds only fail on non-zero exit code so perhaps that is not occurring?